scripts/fkelonthebeat-HNG-06361.js

```diff
@@ -1,11 +1,5 @@
-// Try edit message
-
-var name = "Ochiagha Fredrick";
-var hng_id = "HNG-06361";
-var email = "chidiebubeochiagha31@gmail.com";
-var language = "javascript";
-var stage_No = 2;
-
-
-
-console.log('Hello, world this is ' + name + ' with HNGi7' + ' ID' + hng_id + ' and email ' + email + ' using ' + language + ' for stage ' + stage_No + ' task ');
+let fullName = "Fredrick Ochiagha";
```

Comment:
hey @fkelonthebeat, please change your variable 'fullName' to 'name' instead. Otherwise, all the rest look fine.
Something like 'let name = "Fredrick Ochiagha"

Reply:
Okay